lerna.json

```diff
@@ -1,6 +1,12 @@
 {
   "lerna": "2.9.1",
-  "packages": ["packages/*", "examples/*", "docs", "sandbox/*"],
+  "packages": [
+    "benchmark",
+    "docs",
+    "examples/*",
+    "packages/*",
+    "sandbox/*"
```

Comment:
While I was changing this file, I have sorted the entries in lexicographical order.